<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ViewDialog</class>
 <widget class="QDialog" name="ViewDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>506</width>
    <height>340</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create New View</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/icons/view.png</normaloff>:/icons/view.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="_ignore_grid_layout">
     <item row="2" column="0">
      <widget class="QLabel" name="_ignore_label_3">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Filter With</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QComboBox" name="cb_sort_by"/>
     </item>
     <item row="3" column="3">
      <widget class="QComboBox" name="cb_sorting_order">
       <item>
        <property name="text">
         <string>Ascending Order</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>:/icons/ascending.png</normaloff>:/icons/ascending.png</iconset>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Descending Order</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>:/icons/descending.png</normaloff>:/icons/descending.png</iconset>
        </property>
       </item>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="_ignore_label_5">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Sort By</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="_ignore_label_4">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>View Name</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" colspan="3">
      <widget class="QLabel" name="lb_source">
       <property name="text">
        <string>Table [ Source Table Name ]</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="3">
      <widget class="QLineEdit" name="le_view_name"/>
     </item>
     <item row="3" column="2">
      <widget class="QLabel" name="_ignore_label_6">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Sorting Order</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="_ignore_label">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>View Of</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1" colspan="3">
      <widget class="LineEdit" name="le_formula">
       <property name="placeholderText">
        <string>Leave it empty to select all.</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="_ignore_label_2">
     <property name="text">
      <string>Select Columns</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="_ignore_horizontal_layout_2">
     <item>
      <widget class="QTableWidget" name="table_widget">
       <column>
        <property name="text">
         <string>Selected</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Name</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Data Type</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Display Name</string>
        </property>
       </column>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="_ignore_vertical_layout">
       <item>
        <widget class="QPushButton" name="btn_move_up">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Move Up     </string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>:/icons/arrow up.png</normaloff>:/icons/arrow up.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_move_down">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Move Down</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>:/icons/arrow-down.png</normaloff>:/icons/arrow-down.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_select_all">
         <property name="text">
          <string>Select All</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_deselect_all">
         <property name="text">
          <string>Deselect All</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_invert_selection">
         <property name="text">
          <string>Invert Selection</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="_ignore_vertical_spacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="_ignore_horizontal_layout">
     <item>
      <spacer name="_ignore_horizontal_spacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btn_create_view">
       <property name="text">
        <string>Create View</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="_ignore_btn_cancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LineEdit</class>
   <extends>QLineEdit</extends>
   <header>LineEdit.hpp</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>_ignore_btn_cancel</sender>
   <signal>clicked()</signal>
   <receiver>ViewDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>360</x>
     <y>382</y>
    </hint>
    <hint type="destinationlabel">
     <x>205</x>
     <y>202</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
